src/main/controllers/UserResultsController.ts

```diff
@@ -25,7 +25,8 @@ export class UserResultsController extends RootController {
 
     if (users) {
       if (users.length === 1) {
-        const user = users[0];
+        const user = await req.scope.cradle.api.getUserById(users[0].id);
```

Comment:
Why do we need to make an extra call to the API to get user by ID? The user information is already there after searchForUser() which calls out to the API. Does this getUserById() call return extra information?

Reply:
Yes, as requested by @nikola-naydenov-hmcts. The API will need to make a second call to ForgeRock for populating information regarding the account locked status, with this being limited to calls made to v1/users/:userId, so only a call to that endpoint will return all the information we need for SIDM-6938